src/search/ui/place.directive.html

```diff
@@ -21,8 +21,21 @@
     <div class="udb-place-city" ng-bind="event.address.addressLocality"></div>
   </div>
 
-  <div class="col-sm-2">
-    <span>permanent</span>
+  <div class="col-sm-2" ng-switch="event.calendarType">
+    <span ng-switch-when="permanent">permanent</span>
+    <span ng-switch-when="single">
+        <span class="udb-start-date" ng-bind="event.startDate | date: 'dd/MM/yyyy'"></span>
+    </span>
+    <span ng-switch-when="periodic" class="udb-date-range">
+      <span class="udb-start-date" ng-bind="event.startDate | date: 'dd/MM/yyyy'"></span>
+        <i class="fa fa-long-arrow-right"></i>
+        <span class="udb-end-date" ng-bind="event.endDate | date: 'dd/MM/yyyy'"></span>
+    </span>
```

Comment:
In the case of `periodic`, the ticket mentioned to print the text "periodiek open".

Reply:
Okay I see now in another comment that you discussed with Corneel to keep it consistent with events 👍